### PR TITLE
Added God's Wrath to chaplain's traitor uplink

### PR DESCRIPTION
**Pull request by Arzion5**
:cl:
add: Added God's Wrath, an extremely powerful manifestation of a particularly vengeful god's strength and energy. Similar to the God Hand, but extremely dangerous and conspicuous. You'll know it when you see it. Available at 20 TC!
/:cl:

Chaplains haven't had any role restricted uplink items since the rebase. They used to have a lighting chaplain armor, but that armor hasn't been ported back yet. In the mean time, have a very powerful version of the God Hand!

### CODE REVIEW

**Comment by PopNotes** [comment]
Pretty tasty.

File: hippiestation/code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -28,4 +28,22 @@
...
+	force = 40
+	throw_speed = 3
+	throw_range = 14
+	throwforce = 180 //What the fuck?
+	armour_penetration = 100 //Downs people in 3 hits, the weapon is extremely visible.
+	w_class = WEIGHT_CLASS_HUGE
+	flags_1 = ABSTRACT_1 | NODROP_1 | DROPDEL_1
+	hitsound = 'sound/magic/clockwork/ratvar_attack.ogg'
+	damtype = BURN
+	attack_verb = list("destroyed", "decimated", "wrecked", "punished", "judged")
```
Comment (ghost) [inline hippiestation/code/game/objects/items/weaponry.dm]: PLEASE DON'T USE DECIMATE
it's an abused word that people use thinking it means something else

Comment (KayeArray) [reply hippiestation/code/game/objects/items/weaponry.dm]: # calm down there

File: hippiestation/code/game/objects/items/weaponry.dm [hunk 2]
```diff
@@ -28,4 +28,22 @@
...
+
+/obj/item/syndicate/godhand
+	name = "God's Wrath"
+	desc = "A particularly vengeful god has granted this chaplain its own power."
+	icon_state = "disintegrateblue"
+	item_state = "disintegrateblue"
+	lefthand_file = "hippiestation/icons/mob/inhands/lefthand"
+	righthand_file = "hippiestation/icons/mob/inhands/righthand"
+	force = 40
+	throw_speed = 3
```
Comment (KayeArray) [inline hippiestation/code/game/objects/items/weaponry.dm]: can you not needlessly assign values to these throwing vars please, it's not ever going to be throwable.

File: hippiestation/code/game/objects/items/weaponry.dm [hunk 3]
```diff
@@ -28,4 +28,22 @@
...
 		playsound(src, 'sound/arcade/mana.ogg', 50, 1)
-	..()
\ No newline at end of file
+	..()
+
+/obj/item/syndicate/godhand
+	name = "God's Wrath"
+	desc = "A particularly vengeful god has granted this chaplain its own power."
+	icon_state = "disintegrateblue"
+	item_state = "disintegrateblue"
```
Comment (KayeArray) [inline hippiestation/code/game/objects/items/weaponry.dm:37]: i'm assuming this is a different icon to the one above?

**Review by KayeArray — DISMISSED** [review]

File: hippiestation/code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (PopNotes) [reply hippiestation/code/game/objects/items/weaponry.dm]: yeah, use obliterate instead

**Comment by calzilla1** [comment]
Why would someone get this instead of a d-esword?

**Comment by KayeArray** [comment]
Higher damage, armour penetration, cannot be dropped.
Plus it's thematic.

File: hippiestation/code/game/objects/items/weaponry.dm (hunk 3 above)
Comment (Arzion5) [reply hippiestation/code/game/objects/items/weaponry.dm:37]: I have no idea

**Review by KayeArray — DISMISSED** [review]
day of the rope soon for a (((certain))) bot

**Comment by Arzion5** [comment]
The item also costs 20 TC to off-set the visibility and power.

**Comment by calzilla1** [comment]
Yeah, but it cant refelect tazers nor can it be consealed

File: hippiestation/code/game/objects/items/weaponry.dm [hunk 4]
```diff
@@ -28,4 +28,19 @@
...
+	lefthand_file = "hippiestation/icons/mob/inhands/lefthand"
+	righthand_file = "hippiestation/icons/mob/inhands/righthand"
+	force = 40
+	armour_penetration = 100 //Downs people in 3 hits, the weapon is extremely visible.
+	w_class = WEIGHT_CLASS_HUGE
+	flags_1 = ABSTRACT_1 | NODROP_1 | DROPDEL_1
+	hitsound = 'sound/magic/clockwork/ratvar_attack.ogg'
```
Comment (PopNotes) [inline hippiestation/code/game/objects/items/weaponry.dm]: Eeeeeeeehhhhhhhhhhhhh

File: hippiestation/code/modules/uplink/uplink_item.dm [hunk 5]
```diff
@@ -240,13 +240,20 @@
...
-	
+
 /datum/uplink_item/stealthy_weapons/romerol_kit
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/traitor)
+
+/datum/uplink_item/role_restricted/godhand
+	name = "God's Wrath"
+	desc = "Gain the favor of a particularly vengeful god, granting you its strength and energy concentrated in one hand."
+	item = /obj/item/syndicate/godhand
+	cost = 20
```
Comment (PopNotes) [inline hippiestation/code/modules/uplink/uplink_item.dm]: Could I convince you to go for 18 instead of 20?

**Comment by ghost** [comment]
Sure why not?
It's super expensive but might be fun. +1

**Comment by dreamcrab** [comment]
im stupid wtf does this do

**Comment by Arzion5** [comment]
Traitor item for chaplain

**Comment by dreamcrab** [comment]
U DONT SAY

what possibly could this item do my friendly friend

**Comment by Arzion5** [comment]
It's an item that acts similarly to the god hand, is extremely visible, and deals a massive amount of damage per hit. Also has armor penetration and costs most of the chaplain's TC supply.

**Comment by dreamcrab** [comment]
wtf is a god hand

**Comment by PopNotes** [comment]
a dead gamemode

**Comment by Arzion5** [comment]
The godhand is a chaplain null rod reskin that deals the same damage as a null rod, but burn damage instead of brute damage. Additionaly, it cannot be dropped

**Comment by KayeArray** [comment]
+1

**Comment by ghost** [comment]
Conflicts, stale. Make a new PR when you fix this.

**Pull request closed without merging**